README.md

```diff
@@ -1 +1,148 @@
+# 📗 Table of Contents
 
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements))
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse!] <a name="about-project"></a>
+
+**[Hello Microverse]** is my first project of the Microverse Program which includes creating linters, branching, monitoring github workflows and editing the readme file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. Linters
+2. HTML
+3. CSS
+
+### Tech Stack <a name="tech-stack"></a>
+1. Linters
+2. HTML
+3. CSS
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://stylecss.org/">CSS</a></li>
+    <li><a href="https://linters.org/">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+1. Added Linters.yml file.
+2. Added index.html & style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Live-Demo will be added soon. 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [OPTIONAL] This is supposed to be a mandatory change. I will request it as optional because the previous reviewer did not point it out to you. If your project is not deployed, you should delete the **live demo section**. This applies to the **deployment section** too. Do not write **coming soon**, I suggest you include them in future features 😄

Reply:
Okay, got it. Thanks for the comment.